### PR TITLE
feat: [FFM-6988]: Exposing useLocalStorage and MonacoDiffEditor to MFE

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

exposing custom hook useLocalStorage and MonacoDiffEditor to FF MFE

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/FFCustomMicroFrontendProps.types.ts [hunk 1]
```diff
@@ -46,6 +47,7 @@ export interface FFCustomMicroFrontendProps {
     useQueryParamsState: typeof useQueryParamsState
     usePreferenceStore: typeof usePreferenceStore
     useSyncedEnvironment: typeof useSyncedEnvironment
+    useLocalStorage: typeof useLocalStorage
```
Comment (knagurski) [inline src/modules/75-cf/FFCustomMicroFrontendProps.types.ts:50]: Crystal recently alphabetised these, can we stick to that?

File: src/modules/75-cf/FFCustomMicroFrontendProps.types.ts [hunk 2]
```diff
@@ -55,6 +57,7 @@ export interface FFCustomMicroFrontendProps {
     GitSyncForm: typeof GitSyncForm
     RbacOptionsMenuButton: typeof RbacOptionsMenuButton
     RBACTooltip: typeof RBACTooltip
+    MonacoDiffEditor: typeof MonacoDiffEditor
```
Comment (knagurski) [inline src/modules/75-cf/FFCustomMicroFrontendProps.types.ts:60]: Here too :-)

File: src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx [hunk 3]
```diff
@@ -54,7 +55,8 @@ const FFUIApp: FC = () => (
       useQueryParams,
       useQueryParamsState,
       useSyncedEnvironment,
-      usePreferenceStore
+      usePreferenceStore,
+      useLocalStorage
```
Comment (knagurski) [inline src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx:59]: You get the picture 😉

File: src/modules/75-cf/FFCustomMicroFrontendProps.types.ts (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/FFCustomMicroFrontendProps.types.ts]: Sure, didn't realise

**Review by crystal-fong — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
🔥

**Pull request merged**